packages/lib-react-components/src/MovableModal/MovableModal.js

```diff
@@ -0,0 +1,111 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import { Rnd } from 'react-rnd'
+import ResizeIcon from './components/ResizeIcon'
+import withLayer from '../helpers/withLayer'
+import { Modal } from '../Modal'
+
+function MovableModal (props) {
+  const {
+    children,
+    closeFn,
+    headingBackground,
+    pad,
+    rndProps: {
+      minHeight,
+      minWidth,
+      position,
+      ...restRndProps
```

Comment:
Out of curiosity, where does this `...restRndProps` come from?

Reply:
That's any other props that might be coming in with `rndProps` destructured.